core_lib/src/util/fileformat.h

```diff
@@ -63,6 +63,8 @@ GNU General Public License for more details.
 #define PFF_DEFAULT_PALETTE_EXT \
     QString(".xml")
 
+#define PFF_PALETTE_EXT_FILTER \
```

Comment:
This define is overriding the one from master and should be removed. It's redundant since the macro is defined above.

Reply:
Removed